libmscore/figuredbass.cpp

```diff
@@ -1186,7 +1186,7 @@ qDebug("FiguredBass: duration indicator end line not implemented");
 void FiguredBass::draw(QPainter* painter) const
       {
       // if not printing, draw duration line(s)
-      if (!score()->printing() ) {
+      if (!score()->printing() && !score()->showUnprintable()) {
```

Comment:
Mmm... After further checking... It means that the line will be printed if showUnprintable is false. That's not correct... I will fix it in another commit....